css/style.css

```diff
@@ -5,7 +5,293 @@
 *   light: #fff;
 */
 
+/* Settings */
+
+html {
+  scroll-behavior: smooth;
+}
+
+#wrapper {
+	width: 100%;
+	position: relative;
+	overflow: hidden;
+}
+
 body {
   font-family: 'Montserrat', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+main {
+  position: relative;
+}
+
+img {
+	display: inline-block;
+	vertical-align: top;
+	max-width: 100%;
+	height: auto;
+	border: 0;
 }
 
+ul {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+li {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+/* typography-settings  */
+
+h1,h2,h3,h4,h5,h6,p {
+	margin: 0;
+}
+
+h1 {
+  font-size: 32px;
+  color: #363636;
+}
+
+@media screen and (max-width: 1279px){
+  h1 {
+    font-size: 26px;
+  }
+}
+
+h2 {
+  font-size: 26px;
+  color: #363636;
+}
+
+h4 {
+  font-size: 14px;
+  color: #363636;
+}
+
+p {
+  font-size: 14px;
+  color: #363636;
+  margin: 15px 0;
+}
+
+a {
+  display: block;
+  text-decoration: none;
+  font-size: 14px;
+  color: #363636;
+  font-weight: bold;
+  margin: 15px 0;
+}
+
+@media screen and (max-width: 767px) {
+  a {
+    font-size: 12px;
+    margin: 8px 0;
+  }
+}
+
+a:hover {
+  color: #0078aa;
+  transition: all 0.3s ease-in-out;
+}
+
+/* button-styles */
+
+.button {
+  display: flex;
+  font-size: 30px;
+  cursor: pointer;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+  border: none;
+  background-color: #0078aa;
+  margin-right: 20px;
+}
+
+@media screen and (max-width: 767px) {
```

Comment:
In the case of css (not scss) there is a practice to write one `@media` in the end of file. The `@media` should contain all styles. But you can investigate this moment and share that knowledge with me). I see this kind of organizing medias for the first time, but still it looks reasonable for me.